doc/api/module.md

```diff
@@ -80,15 +80,27 @@ isBuiltin('fs'); // true
 isBuiltin('wss'); // false
 ```
 
-### `module.register()`
+### `module.register(specifier[, parentURL[, options]])`
 
 <!-- YAML
-added: REPLACEME
+added: v20.6.0
```

Comment:
Unrelated change?
```suggestion
added: REPLACEME
```

Reply:
It’s the way I had it already in https://github.com/nodejs/node/pull/49185/files#diff-4e8f3cce79719e4a337f58575b20c998b093eb64164b847ca0eb9ba884d8a801R86.

Reply:
So what? We’re targeting `main`, not the 20.6.0 proposal.